webrtc-stats.html

```diff
@@ -4512,13 +4512,14 @@ <h2>
                 </p>
               </dd>
               <dt>
-                <dfn><code>protocol</code></dfn> of type <span class=
+                <dfn><code>relayProtocol</code></dfn> of type <span class=
                 "idlMemberType">DOMString</span>
               </dt>
               <dd>
                 <p>
-                  Valid values for transport is one of <code>udp</code> and <code>tcp</code>. Based
-                  on the "transport" defined in [RFC5245] section 15.1.
+                  It is the protocol used by the endpoint to communicate with the ICE server.
+                  Valid values are <code>udp</code>, <code>tcp</code>, or <code>tls</code>.
+                  This is the same value that is used for the relay protocol of local ICE candidates.
```

Comment:
What is the reference for "the relay protocol of local ICE candidates"?
Do we have a field in [[WEBRTC]] for it? Or is it an ICE-defined property? If so, which RFC?

Reply:
@fippo could you weigh in...

Reply:
missed that This is an internal reference to https://w3c.github.io/webrtc-stats/#dom-rtcicecandidatestats-relayprotocol

Reply:
Fixed